modules/imaging/Module.php

```diff
@@ -0,0 +1,73 @@
+<?php
```

Comment:
All this module need to move to vertical-imaging-backend  repo - https://github.com/israeli-moh/vertical-imaging-backend (in modules folder)
This is need installed only for imaging installation and not in this repository that contains code for all the verticals